test/Microsoft.Azure.Functions.Worker.Extensions.OpenApi.Tests/OpenApiHttpTriggerContextTests.cs

```diff
@@ -39,7 +40,7 @@ public async Task Given_Type_When_Initiated_Then_It_Should_Return_ApplicationAss
         public async Task Given_Type_With_Referenced_Project_When_Initiated_Then_It_Should_Return_ApplicationAssemblyOfRootAssembly()
         {
             var location = new FileInfo(Assembly.GetExecutingAssembly().Location).Directory.FullName;
-            var context = new OpenApiHttpTriggerContext();
+            var context = new OpenApiHttpTriggerContext(null);
```

Comment:
If we change the constructor parameter to be optional, we don't need to change this part.